### DISCUSSION TITLE
Which Steel Ball Did You Use

### DISCUSSION THREAD

**Discussion by SeanPluemer** [rdudhagra/Sand-Table#2] [Q&A] [answered]
Hello,

This is an amazing post, thank you so much for it. I have been in the process of building my own and this has cleared up many issues/questions I had.
I will most likely reach out in the future for further questions I have, but I can't seem to find in any of the documentation of what steel ball you are using for making the actual patterns. 
I have seen most people use a 1/2 in steel ball. Any inputs? 

Thanks,
Sean

**Comment by rdudhagra** [comment, marked as answer]
Hi Sean,

I’m glad you found this to be helpful! I must have completely forgotten about the marble when I wrote the build guide...apologies for that. I used a steel marble around 9/16” in diameter. I think a half inch ball bearing would work well—it’s a good balance between detail and speed for a table similar in size to mine, although I wouldn’t go smaller than that.

Let me know if you have any more questions, and good luck with your build!
